configs/msc-sm2-imx6dl-ultimain4.2_config

```diff
@@ -1534,7 +1534,7 @@ CONFIG_MTD_M25P80=y
 # CONFIG_MTD_LPDDR2_NVM is not set
 CONFIG_MTD_SPI_NOR=y
 # CONFIG_MTD_MT81xx_NOR is not set
-CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
+# CONFIG_MTD_SPI_NOR_USE_4K_SECTORS is not set
```

Comment:
Is it related to the hardware change? If yes, how does it affect our current SOMs already on the field?

Reply:
There is a check in the jedi-system-update that will handle the printer running the previous kernel, which has the SPI partitioned and with 4K blocks. 
The change from 4K to 64K blocks is only to speed up the erase process.